src/_includes/docs/add-to-app/ios-project/embed-cocoapods.md

```diff
@@ -54,6 +54,15 @@ This section presumes you called your Swift app `MyApp`.
    navigate to the root of your app directory.
    Use the `pod init` command to create the `Podfile` file.
 
+   :::warning
+   CocoaPods 1.15.2 does not support Xcode 16.
```

Comment:
@sfshaza2, I know this doesn't follow the "timeless documentation" best practice. I wasn't sure how to best phrase this. Let me know if you have suggestions!

Reply:
I think it's fine if this situation is temporary, perhaps:

```suggestion
   CocoaPods (as of version 1.15.2) does not support Xcode 16.
```

That also accounts for versions of Cocoapods before 1.15.2.